src/App.tsx

```diff
@@ -47,7 +47,11 @@ const App: FC = () => {
 
   useEffect(() => {
     if (CLOUD && isFlagEnabled('rudderstackReporting')) {
-      load(RUDDERSTACK_WRITE_KEY, RUDDERSTACK_DATA_PLANE_URL)
+        try {
+            load(RUDDERSTACK_WRITE_KEY, RUDDERSTACK_DATA_PLANE_URL)
+        } catch (e) {
```

Comment:
```suggestion
        } catch {
```

That should pass the linter

Reply:
Thanks!